CHANGELOG.md

```diff
@@ -6,6 +6,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.8.0] - Dec, 9, 2021
+
+### Core
+
+#### Added
+- [Icons] New Icons: Icon set 12/2021
+
+### React Components
+
+#### Added
+- [Icons] New Icons: Icon set 12/2021
```

Comment:
I think this could be added to HDS Core also

Reply:
- [ ] add icons to Core as well